### ISSUE TITLE
Javascript error on LMS

### ISSUE THREAD

**Issue by carsongee** [jazkarta/obsolete-edx-sga#1]
I just updated to your last commit, and it looks great in studio, but on the LMS side it appears to be throwing a javascript error that is preventing it from rendering.
https://sandbox0r.mitx.mit.edu/courses/MITx/edx4edx/edx4edx/courseware/Assessment_Problems_chapter/Sample_Problems_sequential/

Is where we have the xblock loaded, error from console in Chrome is:

```
Unable to load Sequence: undefined is not a function lms-modules.d2077e0812e6.js:991
XBlockToXModuleShim lms-modules.d2077e0812e6.js:991
XBlock.initializeBlock lms-application.3b7d87883d98.js:129
(anonymous function) lms-application.3b7d87883d98.js:146
(anonymous function) lms-main_vendor.a86035d2aa44.js:536
e.extend.map lms-main_vendor.a86035d2aa44.js:536
e.fn.e.map lms-main_vendor.a86035d2aa44.js:536
XBlock.initializeBlocks lms-application.3b7d87883d98.js:145
Courseware.Courseware.render lms-courseware.3ebdb45e524f.js:20
Courseware lms-courseware.3ebdb45e524f.js:12
Courseware.Courseware.start lms-courseware.3ebdb45e524f.js:16
(anonymous function) lms-application.3b7d87883d98.js:717
o lms-main_vendor.a86035d2aa44.js:536
p.fireWith lms-main_vendor.a86035d2aa44.js:536
e.extend.ready lms-main_vendor.a86035d2aa44.js:536
c.addEventListener.B
```

I don't have that error in Firefox, but it still doesn't render.

**Comment by chrisrossi** [comment]
Yes, it appears that the javascript environment in studio is vastly different from the one in LMS.

**Comment by chrisrossi** [comment]
Fixed by 4d23f2b22a294ac6dd3b227d1212d4b3217080fa
